.github/workflows/crowdin-download.yml

```diff
@@ -29,3 +29,7 @@ jobs:
           translation: '/**/locales/%two_letters_code%/%original_file_name%'
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Remove last empty lines from translations
+        run: |
+          find '/**/locales/' -type f -name '*.po' -exec sed -i -e '${/^$/d;}' {} \;
```

Comment:
have you tested this? if we add here how it will remove after the file is being pushed on previous step?

Reply:
Yes, I think instead of creating a pull request during the `Download from Crowdin ⬇️` step, we should be creating a pull request at the end.

Reply:
Can you fix the workflow?